org.opent2t.sample.thermostat.superpopular/com.wink.thermostat/js/thingTranslator.js

```diff
@@ -22,13 +24,18 @@ function findResource(schema, di, resourceId) {
         return d.di === di;
     });
 
-    if (!entity) throw new Error('NotFound');
+    if (!entity) {
+        throw new OpenT2TError(404, OpenT2TConstants.DeviceNotFound);
```

Comment:
>OpenT2TError(404, OpenT2TConstants.DeviceNotFound); [](start = 18, length = 51)

The error messages for findResource() is not consistent. In other translators (lamp and binary switch), we have messages like